src/auto/invisible.rs

```diff
@@ -0,0 +1,78 @@
+// This file was generated by gir (38add47) from gir-files (469db10)
+// DO NOT EDIT
+
+use Buildable;
+use Widget;
+use ffi;
+use gdk;
+use glib;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct Invisible(Object<ffi::GtkInvisible, ffi::GtkInvisibleClass>): Widget, Buildable;
+
+    match fn {
+        get_type => || ffi::gtk_invisible_get_type(),
+    }
+}
+
+impl Invisible {
+    pub fn new() -> Invisible {
+        assert_initialized_main_thread!();
+        unsafe {
+            Widget::from_glib_none(ffi::gtk_invisible_new()).downcast_unchecked()
+        }
+    }
+
+    pub fn new_for_screen(screen: &gdk::Screen) -> Invisible {
+        assert_initialized_main_thread!();
+        unsafe {
+            Widget::from_glib_none(ffi::gtk_invisible_new_for_screen(screen.to_glib_none().0)).downcast_unchecked()
+        }
+    }
+}
+
+impl Default for Invisible {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub trait InvisibleExt {
+    fn set_screen(&self, screen: &gdk::Screen);
```

Comment:
Strange, but no `get_screen` here

Reply:
https://developer.gnome.org/gtk3/stable/GtkInvisible.html#gtk-invisible-set-screen ?

Reply:
I meant that https://developer.gnome.org/gtk3/stable/GtkInvisible.html#gtk-invisible-get-screen not generated.

Reply:
The strange thing is that it *is* in the gir file.

Reply:
Ok, I added the method inside a manual trait. But still strange...